awscli/formatter.py

```diff
@@ -52,7 +52,8 @@ def _flush_stream(self, stream):
 
 
 class FullyBufferedFormatter(Formatter):
-    def __call__(self, operation, response, stream=None):
+    def __call__(self, command_name, response,
+                 is_response_paginated=False, stream=None):
```

Comment:
Cleaning it up would be nice. I thought the ``is_response_paginated`` interface was a little awkward when I was updating the emr commands that was using it.

I do not have a preference on what check you make. I feel that checking for ``build_full_response`` would be fine since you call the method within the if statement checking if you can paginate. That way you can avoid an extra import as well for ``isinstance``